src/app/modules/core/manager/landlord/components/view/components/container/container.component.ts

```diff
@@ -9,17 +10,17 @@ import { Landlord } from '@shared/models/landlord'
 export class ViewContainerComponent implements OnInit {
   @Input() landlord: Landlord
   @Output() remove = new EventEmitter<Landlord>()
+  @Output() update = new EventEmitter<Landlord>()
 
-  constructor() {}
+  constructor(private landlordService: LandlordService) {}
 
   ngOnInit(): void {}
 
   toggleStatus(operation: string): void {
     if (operation === 'deactivate' || operation === 'activate') {
-      // chiamata al servizio
       this.landlord.status = !this.landlord.status
+      this.landlordService.toggleStatus(this.landlord)
```

Comment:
A parte il fatto che non avresti dovuto usare un servizio in un presentational component, il servizio sarebbe dovuto essere stato chiamato prima di modificare lo stato sulla UI perche' se la chiamata al servizio ha errori, sulla UI rimarrebbe, erroniamente, il cambio sulla UI.

Reply:
modificato